lib/drivers/docker/container.js

```diff
@@ -0,0 +1,277 @@
+'use strict';
+
+var _ = require('lodash');
+var debug = require('debug')('strong-pm:docker:container');
```

Comment:
I'm using the sort lines plugin in Atom.. at one point it was sorting correctly, but it doesn't seem to any more :-(